hack/eks

```diff
@@ -110,7 +110,7 @@ function load() {
   nodes=$(KUBECONFIG=$RUN_PATH/kubeconfig kubectl get nodes -o json | jq -r '.items[] | .status | .addresses[] | select(.type == "ExternalIP") |.address')
   for node in $(echo $nodes); do
     echo load $image to $node
-    docker save $image | bzip2 | pv | ssh -oStrictHostKeyChecking=no ec2-user@$node 'docker load'
```

Comment:
if bzip2 is not required, can you update the doc and install_comon.sh to remove bzip2 dependency..

Reply:
Updated it.